webapp/blueprint.py

```diff
@@ -0,0 +1,95 @@
+import flask
+
+
+def jp_website_blueprint():
```

Comment:
no need to put it in a function unless you are planning to pass parameters to it.
Same concept as the app